interpreter/text/parser.mly

```diff
@@ -161,6 +192,7 @@ let inline_type_explicit (c : context) x ft at =
 %token INPUT OUTPUT
 %token EOF
 
+%token<string> SIMD_SHAPE
```

Comment:
Can you introduce a proper Vector.shape enum to represent this?

Reply:
Done in newest commits.